routes/cube_routes.js

```diff
@@ -1427,7 +1427,7 @@ function writeCard(res, card, maybe) {
   res.write(`${imgBackUrl},"`);
   card.tags.forEach((tag, tagIndex) => {
     if (tagIndex !== 0) {
-      res.write(', ');
+      res.write(';');
```

Comment:
It might be a good idea to also prevent user-made tags from the UI from containing semicolons, otherwise exporting and re-importing a cube will mess things up.

Reply:
I think that's good for a future PR since we don't prevent commas currently.